config/nephe.yml

```diff
@@ -702,6 +702,16 @@ spec:
   selector:
     control-plane: nephe-controller
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nephe-config
+  namespace: nephe-system
+data:
+  nephe-controller.conf: |
+    ResourcePrefix: anp-
```

Comment:
By default it should be commented out and default value should be nephe

```suggestion
    #ResourcePrefix: nephe
```

Reply:
Solved

Reply:
I see that the value is not commented out.

Reply:
Because the nephe is the default value.

Reply:
imo we should add the `-` separator in the `AddResourcePrefix` func or in options.go setting options, since I don't think we should let user customize separators, or it might cause errors if it's some cloud not allowed character.
@reachjainrahul @Anandkumar26 Should we have some checks on the prefix validity in the webhook, like no cloud not allowed character, no special character etc.?